build.py

```diff
@@ -641,6 +664,22 @@ def build(self):
 
 Project.add(Project_glib_openssl())
 
+class Project_graphene(GitRepo, Meson):
+    def __init__(self):
+        Project.__init__(self,
+            'graphene',
+            repo_url = 'https://github.com/ebassi/graphene',
+            fetch_submodules = False,
+            tag = None,
+            dependencies = ['ninja', 'meson', 'pkg-config', 'glib'],
+            )
+
+    def build(self):
+        Meson.build(self)

```

Comment:
I recall we call the build method in gtk using super, for consistency I guess we should do it in the same way?